doc/whats_new/v0.23.rst

```diff
@@ -269,8 +269,7 @@ Changelog
 
 - |Feature| Added :func:`metrics.mean_absolute_percentage_error` metric and
   the associated scorer for regression problems. :issue:`10708` fixed with the
-  PR :pr:`15007` by :user:`Ashutosh Hathidara <ashutosh1919>`. The scorer and
-  some practical test cases were taken from PR :pr:`10711` by
+  PR :pr:`15007` by :user:`Ashutosh Hathidara <ashutosh1919>` and
```

Comment:
It would be useful to refer to the other PR as well (https://github.com/scikit-learn/scikit-learn/pull/10711)

Reply:
@mohamed-ali , I have mentioned it previously, but @ogrisel suggested me to change this [here](https://github.com/scikit-learn/scikit-learn/pull/15007#discussion_r384657350).

Reply:
Ok sorry. The initial credit line by @ashutosh1919 was better then.

Reply:
It's fine @ogrisel. Changed again. cc @mohamed-ali, Please look at the changes. I also need your approval since you were the first one working on this one.